pages/glossary/programmatic-section-of-content.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Programmatic section of content
+key: programmatic-section-of-content
+unambiguous: true
+objective: true
+---
+
+A _programmatic section of content_ is either an [explicit section of content][] or an [implicit section of content][].
+
+If a programmatic section of content contains a [host][], with a [shadow root][] associated to it, or a [browsing context container][]; then the content of the corresponding shadow tree or browsing context is also part of this programmatic section of content.
```

Comment:
"contains" will need to be better qualified. I'm guessing we're looking for a descendant?

Reply:
Not really. Descendant is OK for explicit sections of content, but not for implicit ones…